src/treeCss.css

```diff
@@ -21,6 +21,7 @@ body {
 .nodeRect {
   border: 2px solid black;
   background: transparent;
+  position: absolute;
 }
 
 .nodeReact .text {
```

Comment:
I guess this is not used anymore?

Reply:
probably not, but text will be added in, so can keep it

Reply:
Ahh, maybe it is just a typo? I thought it was something related to react

Reply:
ah you mean that, probably just a left over or typo